# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/ChromeWorker.cpp
# Commit: 59207e959b7c
# Full Hash: 59207e959b7c3476cc21409d61c8dd03d22a816b
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-12-21 21:21:23
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement requestCredentials for Module Workers; r=jonco
#   
#   This weaves credentials through from the private to script creation.
#   
#   Depends on D155567
# ==============================================================================

diff -r 49f18430c465 -r 59207e959b7c dom/workers/ChromeWorker.cpp
--- a/dom/workers/ChromeWorker.cpp	Tue Dec 20 20:56:13 2022 +0000
+++ b/dom/workers/ChromeWorker.cpp	Tue Dec 20 20:56:13 2022 +0000
@@ -21,7 +21,8 @@
 
   RefPtr<WorkerPrivate> workerPrivate = WorkerPrivate::Constructor(
       cx, aScriptURL, true /* aIsChromeWorker */, WorkerKindDedicated,
-      WorkerType::Classic, u""_ns, VoidCString(), nullptr /*aLoadInfo */, aRv);
+      RequestCredentials::Omit, WorkerType::Classic, u""_ns, VoidCString(),
+      nullptr /*aLoadInfo */, aRv);
   if (NS_WARN_IF(aRv.Failed())) {
     return nullptr;
   }